### ISSUE TITLE
Remove internal logs from user

### ISSUE THREAD

**Issue by bilalshaikh42** [reproducible-biomedical-modeling/CRBM-Viz#1723]
**Please describe the bug**
Users are shown log information that is unrelated to the simulations 

**Please describe how to reproduce the bug**
1. Go to the runBiosimulations application and start a job 
2. When complete, view the error log 
3. Error log will contains text such as the following  

```
--2020-12-13 18:45:30--  https://dispatch.biosimulations.dev/run/5fd6a799f802dc34cdf6e7c9/download
Resolving dispatch.biosimulations.dev (dispatch.biosimulations.dev)... 34.123.157.17
Connecting to dispatch.biosimulations.dev (dispatch.biosimulations.dev)|34.123.157.17|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 18735 (18K) [application/octet-stream]
Saving to: '/home/FCAM/crbmapi/nfs/biosimulations/simulations/5fd6a799f802dc34cdf6e7c9/in/BIOMD0000000297.edited-ssa.omex'

     0K .......... ........                                   100%  570K=0.03s

2020-12-13 18:45:30 (570 KB/s) - '/home/FCAM/crbmapi/nfs/biosimulations/simulations/5fd6a799f802dc34cdf6e7c9/in/BIOMD0000000297.edited-ssa.omex' saved [18735/18735]
```

**How should BioSimulations have behaved**
This information should not be displayed as it is implementation information rather than simulation information. 

----------------------------------------------------

@gmarupilla Could you please work this? Create a PR specific to this issue.   

Here is the approach. 

1) Edit the sbatch template to write some sort of separator to the log such as "-----END-BIOSIMULATORS-INTERNAL-LOG------"
after it calls all the setup steps (module loading, downloading file) etc. but before it calls the singularity image.

2) Edit the method that implements the /logs endpoint to only return the lines in the log after the separator when reading the log file. 

The only changes made should be to the `sbatch.service.ts` file in the dispatch-service and the `app.controller.ts` file in the dispatch-api

**Comment by gmarupilla** [comment]
Here's another approach:
We're only getting these unrelated logs while downloading the omex. What if we replace `wget <OMEX URL>` with `wget <OMEX URL> 1>/dev/null 2>&1` ?

I think this should solve our problem?

**Comment by bilalshaikh42** [comment]
I originally thought this was the easier approach. Only problem, this would
prevent us from being able to view the logs internally to debug issues.
Currently, there are a few jobs missing omex files, and the logs are needed
to determine if the download worked or not.

**Comment by jonrkarr** [comment]
This doesn't seem like a critical issue to address. I think the modern approach (e.g., GitHub actions, Read the Docs, Circle CI) is for users to get complete logs that encompass logs related to the application itself. Following this has the added advantage of not requiring any work.

**Comment by jonrkarr** [comment]
To clarify, I don't think this is a bug. In fact, it could be considered a feature. The current logs are a feature in that they show exactly what runBioSimulations did. One could argue this is good for reproducibility.

Building on this, perhaps we should display to users exactly the commands we used to run the simulation. This is inspired by GitHub actions, CircleCI, etc. which display the exact steps they executed. E.g., 
```
docker pull {image}:{tag}
docker run ...
``

**Comment by bilalshaikh42** [comment]
This makes sense. We would need to have some change to the display at
least, as currently even successful downloads appear as red errors. That
was the what initially caught my eye as a potentially confusing message for
the end user

**Comment by jonrkarr** [comment]
I agree with that. This is similar to what I had in mind with #1626.

For now, we could change the color of the error output to reduce the emphasis on stderr that aren't problematic errors.

Another thing that could be done is to store logs separately for each step similar to what GitHub actions does. This would help users make sense of where errors are coming from.

**Comment by jonrkarr** [comment]
I think this will be addressed by #1733 and #1851.
